### PR TITLE
Shuffle around some code to make it easier to read

### PR DESCRIPTION
**Pull request by KentShikama**
Note I haven't changed a single line of code.

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by KentShikama** [comment]
bors try

File: casper/src/main/scala/coop/rchain/casper/Estimator.scala [hunk 1]
```diff
@@ -173,4 +136,40 @@ object Estimator {
...
+    * the messages defined by blockDag.latestMessages
+    */
+  private def replaceBlockHashWithChildren[F[_]: Monad](
+      b: BlockHash,
+      blockDag: BlockDagRepresentation[F],
+      scores: Map[BlockHash, Long]
+  ): F[List[BlockHash]] =
+    for {
+      c <- blockDag.children(b).map(_.getOrElse(Set.empty[BlockHash]).filter(scores.contains))
+    } yield if (c.nonEmpty) c.toList else List(b)
```
Comment (rabbitonweb) [inline casper/src/main/scala/coop/rchain/casper/Estimator.scala]: Consider 
```
 blockDag.children(b).map(_.getOrElse(Set.empty[BlockHash]).filter(scores.contains)).map {
    case c if (c.nonEmpty) => c.toList
    case _            => b
}
```
instead

Comment (KentShikama) [reply casper/src/main/scala/coop/rchain/casper/Estimator.scala]: Will do in the follow up PR

**Pull request merged**